pkg/compose/compare/docker-compose.yml

```diff
@@ -7,21 +7,24 @@ services:
     command: /cockroach/cockroach start-single-node --insecure --listen-addr cockroach1
     volumes:
       - "${COCKROACH_PATH}:/cockroach/cockroach"
+      - "${LIBGEOS_DIR_PATH}:/cockroach/lib"
   cockroach2:
     # We use a docker image mirror to avoid pulling from 3rd party repos, which sometimes have reliability issues.
     # See https://cockroachlabs.atlassian.net/wiki/spaces/devinf/pages/3462594561/Docker+image+sync for the details.
     image: us-east1-docker.pkg.dev/crl-docker-sync/docker-mirror/docker.io/library/ubuntu:xenial-20170214
     command: /cockroach/cockroach start-single-node --insecure --listen-addr cockroach2
     volumes:
       - "${COCKROACH_PATH}:/cockroach/cockroach"
+      - "${LIBGEOS_DIR_PATH}:/cockroach/lib"
   test:
     # We use a docker image mirror to avoid pulling from 3rd party repos, which sometimes have reliability issues.
     # See https://cockroachlabs.atlassian.net/wiki/spaces/devinf/pages/3462594561/Docker+image+sync for the details.
     image: us-east1-docker.pkg.dev/crl-docker-sync/docker-mirror/docker.io/library/ubuntu:xenial-20170214
     environment:
       - COCKROACH_DEV_LICENSE=$COCKROACH_DEV_LICENSE
+      - COCKROACH_RUN_COMPOSE_COMPARE=${COCKROACH_RUN_COMPOSE_COMPARE}
```

Comment:
I am unsure how these tests were running at all before this change. Do you know @rickystewart?

Reply:
Ah, it looks like this is something I missed in `156d9c9c9050df77f38db04a0187503e2edf5cbe`. So the thing has not been running successfully since last week. Thanks for catching this.

Reply:
That means this has to be backported to `release-24.1`.

Reply:
I was planning to backport anyway, since the "missing libgeos library" error is/was happening on all active branches

Reply:
This change won't backport cleanly to `release-23.2` or prior, the basic idea should still apply though.